src/utils/listenerHelpers.ts

```diff
@@ -51,6 +52,11 @@ export function setupDispatchOverride() {
   };
 }
 
+void (async () => {
+  await pollForValue(() => Calc);
+  setupDispatchOverride();
+})();
```

Comment:
This is already in `runDesModder()` in preload/script.ts

Reply:
I've removed it from `runDesModder()` because I noticed that that was causing it to run twice and delete the handler the second time.